rasa/model.py

```diff
@@ -80,6 +84,14 @@ class Section(NamedTuple):
 )
 SECTION_NLG = Section(name="NLG templates", relevant_keys=[FINGERPRINT_NLG_KEY])
 
+SECTION_FINE_TUNE = Section(
+    name="Fine-tune",
+    relevant_keys=[
+        FINGERPRINT_CONFIG_WITHOUT_EPOCHS_KEY,
+        FINGERPRINT_DOMAIN_WITHOUT_NLG_KEY,
```

Comment:
Is this correct to ensure labels have not changed? Maybe there is a more looser definition e.g. `Domain.input_states`?

Reply:
I think it should be correct. I was first confused by the naming of `FINGERPRINT_DOMAIN_WITHOUT_NLG_KEY` as I thought this is the fingerprint of the domain without the `responses:` key but looks like they are also included inside `domain.fingerprint()`.
Maybe @wochinge has a better opinion on it.

Reply:
I'm positively surprised how useful the `Domain` is 😄 In my opinion it's fine!